.circleci/config.yml

```diff
@@ -65,6 +65,12 @@ jobs:
           command: |
             curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
             LOCAL_DIR=./sdk pod install
+      # Install fastlane and setup
+      - run:
+          name: Setup CircleCI
+          command: |
+            bundle install
+            bundle update fastlane
```

Comment:
Do we need to do this? From the job above it looks like fastlane is available by default? Also I think `bundle update fastlane` will fail as the project doesn't contain a Gemfile.

Could we just remove the `bundle exec` from the commands below, to remove the need for a `.bundle` directory to exist?

Reply:
I have added the missing Gem file. This will ensure we are always using the same version of fastlane. The one that is globally installed by circleci is out dated.